uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -254,6 +252,22 @@ define([
       };
 
       /**
+       * Checks if a leave request has been updated since opening the modal
+       *
+       * @return {Boolean}
+       */
+      this.hasRequestUpdated = function () {
+        // angular copy is necessary to remove the $$hashkey property from
```

Comment:
I would put all this comments in the documentation, so they don't pollute the code
```js
/**
 * Checks...
 *
 * Angular copy is..
 * fileUploader...
 *
 * @return {Boolean}
 */
```